dot/sync/utils.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package sync
```

Comment:
Maybe move this as a subpackage to `dot/sync`? Since we want to split things in subpackages 🤔

Reply:
This is just a single file. I am not sure if we do that. 🤔 @qdm12

Reply:
Maybe have it as an `outlier` subpackage? We definitely have too big single files (not this one though, yet) and too many files in some packages. It's also discouraged to have generic names such as `utils` since it doesn't mean anything specific. At least have it as e.g. `outlier.go`? Although creating subpackages when it feels appropriate is good too.

Reply:
changed into outliers.go ✔